### PR TITLE
layouts: fix tcl with clientcount of 2

### PR DESCRIPTION
**Pull request by uvera**

### CODE REVIEW

**Comment by con-f-use** [comment]
Please tell us why this is better and what problem it actually fixes.

**Comment by uvera** [comment]
Two windows don't display properly on TCL

**Comment by con-f-use** [comment]
How so? I don't see a problem on my system.

**Comment by con-f-use** [comment]
Yeah, I just noticed, that on my first monitor, they display correctly, but on my second, they don't.

image

**Comment by con-f-use** [comment]
But just using the `tile` layout in case of two windows isn't a fix in my opinion. The tcl layout should do it correctly on its own.

**Comment by uvera** [comment]
I agree. Will see if I can do something.

**Comment by uvera** [comment]
Think this is it. Now it works fine, tested all scenarios and mfacts.

File: layouts.c [hunk 1]
```diff
@@ -304,7 +304,7 @@ tcl(Monitor * m)
 
 	if (n > 1)
 	{
-		x = m->wx + ((n > 1) ? mw + sw : mw);
```
Comment (uvera) [inline layouts.c:307]: This check should be redudant.

**Pull request merged**